docs/video/01-build/07-Develop-End-an-ongoing-Video-Call.mdx

```diff
@@ -16,26 +16,21 @@ This section covers everything you need to know about ending an ongoing video ca
   <title>Ending an Ongoing Video Call | Push Video | Push Documentation</title>
 </head>
 
-import { ABlock } from "@site/src/css/SharedStyling";
-
 ## Prerequisites
 
 - You have initialized Push Video API. Please refer to the [Initializing Video API](../initializing-video-api) section for more details.
 - You are listening to Video Stream events, Please refer to the [Stream API](../stream-video) section for more details.
+- There is an ongoing video call between the user and a peer.
 
 ## Ending an Ongoing Video Call
 
 When the user decides to end an ongoing video call, we call the `disconnect()` method.
 
 ```typescript
-// aliceVideoCall.disconnect(address);
-await aliceVideoCall.disconnect(address);
+// aliceVideoCall.disconnect();
+await aliceVideoCall.disconnect();
 ```
```

Comment:
refer to chat to ensure parameters and api are segregated as it is in chat docs, also disconnect also has recipients as an optional parameter so options and parameters table should be present